.config/forest.dic

```diff
@@ -33,3 +33,9 @@ tuple
 validator/S
 mempool
 sortition
+struct/S
+serializer/SM
```

Comment:
`serializer` is already in the dictionary.

Reply:
Removing serializer from forest.dic causes the errors to reappear, doesn't look like it is in the dictionary?

Reply:
Merge with `main` and try again.

Reply:
Tried again, same thing

Reply:
Ah, I saw it wrong, only `serialize` is in the dictionary.